lib/cli/commands/create.ts

```diff
@@ -54,13 +54,22 @@ const formatOutput = (response: CreateNodeResponse.AsObject) => {
     }
     console.log('-----------------------END XUD SEED----------------------\n');
 
+    let walletInitializedMessage = 'The following wallets were initialized: ';
+
     if (response.initializedLndsList.length) {
-      console.log(`The following lnd wallets were initialized: ${response.initializedLndsList.join(', ')}`);
+      walletInitializedMessage += response.initializedLndsList.join(', ');
```

Comment:
What if Raiden is not initialized (only lnds)? The message will end with ", ".

Reply:
Not it won't because the [`join` method just puts the separator between the elements of the array](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/join). The message will end with the last entry in `response.initializedLndsList`